JS-test/main.js

```diff
@@ -0,0 +1,34 @@
+//Write a function that logs to the console numbers 1 to 100.
+function counting() {
+  for (let i = 1; i < 101; i++) {
+    if (i % 3 == 0 && i % 5 == 0) {
+      console.log("Jackpot!");
+    } else if (i % 3 == 0) {
+      console.log(" This is divisible by 3");
+    } else if (i % 5 == 0) {
+      console.log("This is divisible by 5");
+    } else console.log(i);
+  }
+}
+
+counting();
+
+//2. (Main theme: DOM manipulation)
+const body = document.querySelector("body");
+const btn = document.createElement("button");
+btn.appendChild(body);
+btn.addEventListener("click", (e) => {
+  document.createElement("img");
+  const img = document.querySelector("img");
+  img.src = "https://picsum.photos/seed/picsum/200/300";
+  img.appendChild(img);
+});
+
+//3. (Main theme: Async API calls)
+// fetch('https://reqres.in/api/users')
+// .then(response => response.json())
+// .then( for (let i=0;i<4;i++){
+//     // const userName=i.first_name;
+//     //const showCase=document.querySelector('api')
+//     // showCase.innerHTML('userName')
+// });
```

Comment:
```suggestion
 .then((users) => { // Here you need to get the JSON
   for (let i = 0; i < 4; i++) {
       const user = users[i]; // Grab a single user
       const userName = user.first_name; // Grab their first name
       const showCase = document.querySelector('api'); // I don't see this in your HTML so make sure it exists
       showCase.innerHTML(userName); // Make sure it's the variable, not a string 'userName'
   }
 });
```

A `.then` handler always has to accept a function, so while the `for` block has `{}` it does not create a function.